### PR TITLE
all translation keys

### PR DESCRIPTION
**Pull request by Josef-Haupt**
* adds multi-language support to GUI

### CODE REVIEW

File: gui.py [hunk 1]
```diff
@@ -1180,7 +1186,7 @@ def build_train_tab():
                             info=loc.localize("training-tab-classifier-textbox-info"),
                         )
                         output_format = gr.Radio(
-                            ["tflite", "raven", "both"],
+                            ["tflite", "raven", (loc.localize("training-tab-output-format-both"),"both")],
```
Comment (Josef-Haupt) [inline gui.py:1189]: The `both` afterwards is unessary, right?

Comment (max-mauermann) [reply gui.py:1189]: I assumed its passed as value, so we need the 'both' so we dont pass the translated value to the script.

File: lang/de.json [hunk 2]
```diff
@@ -4,18 +4,18 @@
     "single-tab-output-header-start": "Start (s)",
     "single-tab-output-header-end": "Ende (s)",
     "single-tab-output-header-sci-name": "Wissenschaftlicher Name",
-    "single-tab-output-header-common-name": "Gebräuchlicher Name",
+    "single-tab-output-header-common-name": "Trivialname",
```
Comment (Josef-Haupt) [inline lang/de.json:7]: Does that word really exist?

File: lang/de.json [hunk 3]
```diff
@@ -31,12 +31,12 @@
     "species-list-coordinates-lon-number-info": "Längengrad des Aufnahmestandorts.",
     "species-list-coordinates-yearlong-checkbox-label": "Ganzjährig",
     "species-list-coordinates-week-slider-label": "Woche",
-    "species-list-coordinates-week-slider-info": "Geben Sie die Woche des Jahres an, in der die Aufnahme gemacht wurde, wobei jeder Monat in vier Wochen unterteilt wird. Wählen Sie einen Wert von 1 bis 48.",
+    "species-list-coordinates-week-slider-info": "Geben Sie die Woche des Jahres an, in der die Aufnahme gemacht wurde. Jeder Monat ist dabei in vier Wochen unterteilt wird. Wählen Sie einen Wert von 1 bis 48.",
```
Comment (Josef-Haupt) [inline lang/de.json:34]: Could maybe remove **dabei**.

File: lang/de.json (hunk 2 above)
Comment (max-mauermann) [reply lang/de.json:7]: yes (https://de.wikipedia.org/wiki/Trivialname)

**Pull request merged**